tdvt/tdvt/exprtests/regexcalcs/setup.string.regexp.extract_nth.extended.txt

```diff
@@ -1,4 +1,55 @@
 REGEXP_EXTRACT_NTH("",'([a-z]+)\s+(\w+)',2)
 REGEXP_EXTRACT_NTH("abcc @#$%^^ abcd abce ",'([a-z]+)\s+(\w+)',2)
 REGEXP_EXTRACT_NTH("abcc @#$%^^ abcd abce ",'([a-z]+)\s+(\w+)',0)
-REGEXP_EXTRACT_NTH("abcc @#$%^^ abcd abce ",'',2)
```

Comment:
ignore this, the "difference" is that this line didn't end with `\n` before; wasn't actually removed